timely_beliefs/beliefs/classes.py

```diff
@@ -318,6 +327,7 @@ def search_session(  # noqa: C901  # todo: remove after removing deprecated argu
         place_beliefs_in_sensor_timezone: bool = True,
         place_events_in_sensor_timezone: bool = True,
         custom_filter_criteria: List[BinaryExpression] = None,
+        custom_join_targets: List[JoinTarget] = None,
```

Comment:
Technically, this (and the other custom parameter above are `Optional`).

And then, the following sentence in the docstring is not correct anymore (now we have something else than filters among them):

"The optional arguments represent optional filters."